.github/workflows/deploy.yml

```diff
@@ -4,241 +4,40 @@ on:
-            - name: Check outputs
-              run: |
-                  echo "Pull Request Number - ${{ env.PULL_REQUEST_NUMBER }}"
-                  echo "Pull Request Number - ${{ steps.cpr.outputs.pull-request-number }}"
-    kovan-staging:
-        runs-on: ubuntu-latest
-        if: github.event.comment.body == 'deploy kovan-staging'
-        steps:
-            - name: Fetch issue info
-              uses: octokit/request-action@v2.x
-              id: get_issue_info
-              with:
-                  route: GET ${{ github.event.issue.url }}
-              env:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            - name: Parse issue info
-              id: parse_issue_info
-              uses: gr2m/get-json-paths-action@v1.x
-              with:
-                  json: ${{ steps.get_issue_info.outputs.data }}
-                  pull_request_url: 'pull_request.url'
-            - name: Fetch PR info
-              uses: octokit/request-action@v2.x
-              id: get_pr_ref
-              with:
-                  route: GET ${{ steps.parse_issue_info.outputs.pull_request_url }}
-              env:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            - name: Parse PR info
-              id: parse_pr_info
-              uses: gr2m/get-json-paths-action@v1.x
-              with:
-                  json: ${{ steps.get_pr_ref.outputs.data }}
-                  ref: 'head.ref'
-                  sha: 'head.sha'
-            - run: 'echo ${{ steps.parse_pr_info.outputs }}'
-            - uses: actions/checkout@v2
-              with:
-                  token: ${{ secrets.PAT }}
-                  repository: 0xProject/0x-main-infra
-            - name: Update API
-              run: sed -i -e 's/"883408475785\.dkr\.ecr\.us-east-1\.amazonaws\.com\/0x\/api\:.*"/"883408475785\.dkr\.ecr\.us-east-1\.amazonaws\.com\/0x\/api\:${{ steps.parse_pr_info.outputs.sha }}"/g' env/0x-api/kovan-staging/common/*.yml env/0x-api/kovan-staging/v4/*.yml
+            - name: Update API configs
+              run: sed -i -e 's/"883408475785\.dkr\.ecr\.us-east-1\.amazonaws\.com\/0x\/api\:.*"/"883408475785\.dkr\.ecr\.us-east-1\.amazonaws\.com\/0x\/api\:${{ github.sha }}"/g' env/0x-api/${{ env.folder }}/common/*.yml env/0x-api/${{ env.folder }}/apis/*.yml
             - name: Create Pull Request
               uses: peter-evans/create-pull-request@v3.5.0
               with:
                   token: ${{ secrets.PAT }}
-                  commit-message: '[0xApi][staging][kovan] Deploy ${{ steps.parse_pr_info.outputs.ref }}'
-                  title: '[0xApi][staging][kovan] Deploy ${{ steps.parse_pr_info.outputs.ref }}'
-                  reviewers: dekz
-                  labels: 0xApi, kovan, staging
-                  branch: '0xApi/kovan-staging/${{ steps.parse_pr_info.outputs.ref }}'
+                  commit-message: '0x-api/${{ env.folder }}: Deploy ${{ github.sha }}'
```

Comment:
Seems very unlikely, but asking just in case: any issue that might arise from the commit message changing?

Reply:
nooop, this is just for us to reference internally when debugging deploys